src/test/java/org/wildfly/security/sasl/util/ServerFactoryHelpersTest.java

```diff
@@ -0,0 +1,74 @@
+package org.wildfly.security.sasl.util;
```

Comment:
I know this is just a move but could we please get a copyright header on this one.